src/main/java/org/example/Mapper/Collections/CollectionsStat.java

```diff
@@ -19,21 +19,42 @@ public String getType() {
         return type;
     }
 
+    public void setType(String type) {
+        this.type = type;
+    }
+
     public boolean isInheritance_enabled() {
         return inheritance_enabled;
     }
 
+    public void setInheritance_enabled(boolean inheritance_enabled) {
+        this.inheritance_enabled = inheritance_enabled;
+    }
+
     public List<Permissions> getPermissions() {
         return permissions;
     }
 
+    public void setPermissions(List<Permissions> permissions) {
+        this.permissions = permissions;
+    }
+
     public boolean isRegistered() {
         return registered;
     }
 
+    public void setRegistered(boolean registered) {
+        this.registered = registered;
+    }
+
     public int getModified_at() {
         return modified_at;
     }
+
+    public void setModified_at(int modified_at) {
+        this.modified_at = modified_at;
+    }
+
     public static class Permissions {
```

Comment:
Data objects and collections have permissions. Can this class be shared?

Open an issue to investigate that. We'll handle that in a separate PR as well.

Reply:
I opened an issue. But I did end up deciding it would be best to not handle anything regarding mapping and leave it up to the user what they want to do with the JSON string.